internal/handlers/pg/pgdb/query_documents_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Use _test package to avoid import cycle with testutil.
+package pgdb_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	"golang.org/x/net/context"
+
+	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+)
+
+func TestQueryDocuments(t *testing.T) {
+	t.Parallel()
+
+	ctx := testutil.Ctx(t)
+
+	pool := getPool(ctx, t, nil, zaptest.NewLogger(t))
+	dbName := testutil.DatabaseName(t)
+	collectionName := testutil.CollectionName(t)
+
+	t.Cleanup(func() {
+		pool.DropDatabase(ctx, dbName)
+	})
+
+	pool.DropDatabase(ctx, dbName)
+	require.NoError(t, pgdb.CreateDatabase(ctx, pool, dbName))
+
+	cases := []struct {
+		name             string
+		collection       string
+		documents        []*types.Document
+		docsPerIteration []int // how many documents should be fetched per each iteration
+		// len(docsPerIteration) is the amount of fetch iterations.
```

Comment:
let's remove it? Unused.

Reply:
@seeforschauer ok, I moved this part of documentation before the `docsPerIteration` definition. Hopefully, it makes more sense now.